gkeys/gkeys/mail.py

```diff
@@ -37,10 +38,11 @@ def __init__(self, login, logger):
     def send_email(self, uid, message):
         self.logger.debug(_unicode('Sending email with message %s') % _unicode(message))
         subject = "Expiring Key"
-        email_to = _unicode("justthisthing@gmail.com") #uid
+        email_to = uid
```

Comment:
Please re-read my previous comments about an optional DEBUG_TARGET that overrides the uid at this point.